src/debug.h

```diff
@@ -53,6 +53,12 @@
 #define __FUNCTION_NAME__ __func__
 #endif
 
+#define fatal(cond) \
+    if( !( cond ) ) { \
+        debugmsg( "Assertion failed: %s", STRING( cond ) ); \
+        std::terminate(); \
```

Comment:
This is not equivalent to assert; it won't tell the user anything if it's triggered before there's a window (e.g. on Windows, where the user doesn't see cerr)